gulpfile.js

```diff
@@ -1,9 +1,10 @@
 "use strict";
 
+import { deleteSync } from 'del';
```

Comment:
The newer `del` needs us to use "import" rather than "require".

But "import" only works if this code is a "module" (see the `"type": "module"` added to `package.json`)
And some discussion in https://stackoverflow.com/questions/58384179/syntaxerror-cannot-use-import-statement-outside-a-module

But now lint CI complains:
```
ReferenceError: require is not defined in ES module scope, you can use import instead
```

The other `require()` in this file stop working. They would also have to be changed to `import` statements.

I will leave this code here as a clue to where I got to in a few minutes.
Probably there is some easier solution to make this work, someone can try to find it if this major version upgrade is needed.